.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: php
 php:
-  - 7.0
-  - 7.1
   - 7.2
   - 7.3
+  - 7.4
```

Comment:
If we're using Github Action now, maybe it's time to remove Travis?

Reply:
Yes, but I would like to see the transition before delete it